src/PCLExt.FileStorage.UWP/StorageFolderImplementation.cs

```diff
@@ -0,0 +1,285 @@
+﻿using PCLExt.FileStorage.UWP.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Search;
+
+namespace PCLExt.FileStorage.UWP
+{
+    sealed class StorageFolderImplementation : IFolder
+    {
+        public string Name => _storageFolder.Name;
+
+        public string Path => _storageFolder.Path;
+
+        public bool Exists => true;
```

Comment:
It can be deleted at any time of the app's execution. Invalid.

Reply:
I'll fix it